src/tests/unit/fixtures/workload/mock_nsx_remediation.py

```diff
@@ -0,0 +1,110 @@
+"""Mock data for NSX Remediation API."""
+
+NSX_REQUEST_1 = {
+    "device_ids": [142, 857],
+    "action_type": "NSX_REMEDIATION",
+    "options": {
+        "toggle": "ON",
+        "tag": "CB-NSX-Quarantine"
+    }
+}
+
+NSX_RESPONSE_1 = {
+    "job_ids": [
+        "7ff05537-350a-420c-bfa8-3408ac70ce53"
```

Comment:
Use a sample ID fetched from a CBC environment as I believe they don't use a raw guid. Please replace all job_ids below

Reply:
The job IDs generated are all in the form `NSX_REMEDIATE_{guid}`.  I have changed the job IDs used in the mocks and unit tests to match this pattern.